src/linear_combination.jl

```diff
@@ -64,6 +64,10 @@ Applies the [`LinearCombination`](@ref) to each of the specified cols in `x`.
 If no `cols` are specified, then the [`LinearCombination`](@ref) is applied to all columns.
 """
 function apply(x, LC::LinearCombination; cols=nothing)
+    if cols isa Union{Symbol, AbstractString}  # want same behaviour for single column
```

Comment:
We can use multiple dispatch here instead wich is generally more performant

```julia
function apply(x, LC::LinearCombination; cols::Union{Symbol, AbstractString})
     apply(x, LC; cols=[cols]
 end
 ```

Reply:
> So you'll need something like a _to_vec method

Just note I'm trying to allow `cols` as a `Tuple`, so I want to dispatch the opposite way on `Union{Symbol, AbstractString}` rather than `AbstractArray`.

Reply:
Maybe I should just do `Union{Tuple, AbstractArray}`

> > So you'll need something like a _to_vec method
> 
> Just note I'm trying to allow `cols` as any collection including `Tuple`, so I want to dispatch the opposite way on `Union{Symbol, AbstractString}` rather than `AbstractArray`.

Maybe just `Union{Tuple, AbstractArray}` is good enough and clearer, actually.